### PR TITLE
refactor: rename mean finance to balmy

### PR DESCRIPTION
**Pull request by 0xsambugs**
Hey Llamas!

Mean Finance [has rebranded](https://twitter.com/balmy_xyz/status/1785324575011512520) to Balmy, so it's time to update some metadata!

I couldn't test this properly, so please let me know if there is something missing (or feel free to fix it yourselvs)

### CODE REVIEW

File: defi/src/protocols/data.ts [hunk 1]
```diff
@@ -11262,31 +11262,38 @@ The eWIT token is a custodial, wrapped version of the Witnet coin managed by the
...
-    audits: "2",
+    logo: `${baseIconsUrl}/balmy.png`,
+    audits: "5",
     audit_note: null,
     gecko_id: null,
     cmcId: null,
     category: "Services",
-    chains: ["Ethereum", "Arbitrum", "Polygon", "Optimism", "BSC"],
+    chains: ["Ethereum", "Arbitrum", "Polygon", "Optimism", "BSC", "Gnosis", "Moonbeam"],
     module: "meanfinance/index.js",
```
Comment (0xsambugs) [inline defi/src/protocols/data.ts:11279]: I'm not sure what this is, so I wasn't sure how to rename it

File: coins/src/adapters/yield/balmy/balmy.ts [hunk 2]
```diff
@@ -5,12 +5,12 @@ import { fetch } from "../../utils";
 
 export default async function getTokenPrices(chain: string, timestamp: number) {
   const { transforms }: { transforms: { type: string; dependent: string }[] } = await fetch(
-    `https://api.mean.finance/v1/transforms/networks/${chain}/transforms`,
+    `https://api.balmy.xyz/v1/transforms/networks/${chain}/transforms`,
```
Comment (Define101) [inline coins/src/adapters/yield/balmy/balmy.ts:8]: can we just keep this. All other changes please remove

File: coins/src/adapters/index.ts [hunk 3]
```diff
@@ -39,7 +39,7 @@ export default {
   yearn: require("./yield/yearn"),
   convex: require("./yield/convex"),
   alchemix: require("./yield/alchemix"),
-  meanFinance: require("./yield/mean-finance"),
+  balmy: require("./yield/balmy"),
```
Comment (Define101) [inline coins/src/adapters/index.ts:42]: no need to rename it

File: coins/src/adapters/yield/balmy/balmy.ts [hunk 4]
```diff
@@ -5,12 +5,12 @@ import { fetch } from "../../utils";
...
   );
   const tokens4626 = transforms
     .filter((t) => t.type === "ERC4626")
     .map(({ dependent }) => dependent);
-  return await unwrap4626(chain, tokens4626, timestamp, "mean-finance");
+  return await unwrap4626(chain, tokens4626, timestamp, "balmy");
```
Comment (Define101) [inline coins/src/adapters/yield/balmy/balmy.ts:13]: keep mean-finance

File: coins/src/adapters/yield/balmy/index.ts [hunk 5]
```diff
@@ -1,6 +1,6 @@
-import getTokenPrices from "./mean-finance";
+import getTokenPrices from "./balmy";
 
-export function meanFinance(timestamp: number = 0) {
+export function balmy(timestamp: number = 0) {
```
Comment (Define101) [inline coins/src/adapters/yield/balmy/index.ts:3]: also remove changes to this file

File: coins/src/adapters/yield/misc4626/index.ts [hunk 6]
```diff
@@ -2,7 +2,7 @@ import getTokenPrices from "./misc";
 import tokens from "./tokens.json";
 import tokensQiDAO from "./tokensQiDAO.json";
 import { getYieldWrites2 } from "../../utils/yieldTokens";
-import { unwrap4626 } from "../mean-finance/mean-finance";
+import { unwrap4626 } from "../balmy/balmy";
```
Comment (Define101) [inline coins/src/adapters/yield/misc4626/index.ts:5]: also remove changes to this file

File: coins/src/adapters/yield/misc4626/misc.ts [hunk 7]
```diff
@@ -1,4 +1,4 @@
-import { unwrap4626 } from "../mean-finance/mean-finance";
+import { unwrap4626 } from "../balmy/balmy";
```
Comment (Define101) [inline coins/src/adapters/yield/misc4626/misc.ts:1]: also remove changes to this file

**Comment by Define101** [comment]
We will make the changes one second

**Comment by Define101** [comment]
https://github.com/DefiLlama/defillama-server/pull/6859

**Comment by Define101** [comment]
https://github.com/DefiLlama/defillama-server/pull/6860

**Comment by 0xsambugs** [comment]
Hey @Define101 , thank you for making the changes!

Overall looks good, except for the logo:
Screenshot 2024-05-09 at 8 26 26 PM

Could I ask for help with that?

**Comment by Define101** [comment]
Logo  should be good within 1 hour
Sorry

**Comment by 0xsambugs** [comment]
Great, thank you @Define101 ! 🙏

**Pull request closed without merging**